### PR TITLE
User intent

### PR DESCRIPTION
**Pull request by vincenthoogsteder**
- first basic user intent analysis
- defined buckets based on our production as in line with the quickstart

### CODE REVIEW

File: notebooks/basic-user-intent.ipynb [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+   "metadata": {},
+   "source": [
+    "## Defining different stages of user intent\n",
+    "After exploring the `root_location` and `session_duration` (both per root location and quantiles), we can make a simple definition of different stages of user intent.\n",
+    "\n",
+    "Based on the objectiv.io website data in the quickstart, we could define them as:\n",
+    "\n",
+    "| User intent | Root locations | Duration |\n",
+    "| :--- | :--- | : --- |\n",
+    "| 1 - Inform | home, blog, about, modeling, taxonomy, tracking | less than 2 minutes |\n",
```
Comment (hendrik-obj) [inline notebooks/basic-user-intent.ipynb:181]: Now changed this to any root for first bucket. Only a restriction on bucket  and 3

File: notebooks/basic-user-intent.ipynb [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dc42e571",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set the root locations that we will use based on the definitions above\n",
+    "roots = bach.DataFrame.from_pandas(engine=df.engine, \n",
+    "                                   df=pd.DataFrame({'roots': ['modeling', 'taxonomy', 'tracking', 'home']}), \n",
```
Comment (hendrik-obj) [inline notebooks/basic-user-intent.ipynb:211]: the old 'docs' is not in this list, so those will be '1 - inform'

File: notebooks/basic-user-intent.ipynb [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+    "## Defining different stages of user intent\n",
+    "After exploring the `root_location` and `session_duration` (both per root location and quantiles), we can make a simple definition of different stages of user intent.\n",
+    "\n",
+    "Based on the objectiv.io website data in the quickstart:\n",
+    "\n",
+    "We think that people that spent most time (>80th percentile) and specifically in our documentation sections are in the Implement phase of Objectiv. If you look at the other quantiles, it feels sensible to deem that users beyond the 50th and 80th perctile, spending 2-5 minutes in our documentation sections are Exploring. The remaining users are informing themselves about the product. Those users are spending less than two minutes in the docs and/or spend any amount of time on our main website.\n",
+    "\n",
+    "| User intent | Root locations | Duration |\n",
+    "| :--- | :--- | : --- |\n",
+    "| 1 - Inform | *all sections other than the ones mentioned below* | *any time spent* |\n",
```
Comment (hendrik-obj) [inline notebooks/basic-user-intent.ipynb:185]: updated this table and the text a bit, have a look. Two rows for inform, so maybe make the columns a bit different to make it prettier.

File: notebooks/basic-user-intent.ipynb (hunk 2 above)
Comment (vincenthoogsteder) [reply notebooks/basic-user-intent.ipynb:211]: good point, let me add that, for the reader it's just about understanding the concept anyway and not specifically the roots

**Pull request merged**